RecoEcal/EgammaCoreTools/interface/EcalClusterTools.h

```diff
@@ -1025,10 +1025,10 @@ std::vector<float> EcalClusterToolsT<noZS>::covariances(const reco::BasicCluster
     covEtaPhi = 0;
     covPhiPhi = 0;
   }
-  std::vector<float> v;
-  v.push_back(covEtaEta);
-  v.push_back(covEtaPhi);
-  v.push_back(covPhiPhi);
+  std::array<float, 3> v;
+  v[0] = covEtaEta;
+  v[1] = covEtaPhi;
+  v[2] = covPhiPhi;
```

Comment:
here and everywhere else
```suggestion
  std::array<float, 3> v{{covEtaEta, covEtaPhi, covPhiPhi}};
```